.travis.yml

```diff
@@ -4,11 +4,19 @@ dist: trusty
 group: edge
 
 before_install:
+    - sudo add-apt-repository ppa:adrozdoff/cmake
     - sudo apt-get -qq update
-    - sudo apt-get install -y libmpfr-dev libgomp1 gcc
-    
+    - sudo apt-get install -y clang cmakelibmpfr-dev libgomp1 gcc
```

Comment:
`cmakelibmpfr-dev` looks like a typo.

Also, please remove clang for now. Let's make it work with one compiler, then extend it.